gsa/src/web/pages/reports/detailspage.js

```diff
@@ -96,7 +96,7 @@ const DEFAULT_FILTER = Filter.fromString(
   'levels=hml rows=100 min_qod=70 first=1 sort-reverse=severity',
 );
 
-const REPORT_FORMATS_FILTER = Filter.fromString('active=1 and trust=1 rows=-1');
+const REPORT_FORMATS_FILTER = Filter.fromString('active=1 and trust=1');
 
```

Comment:
This doesn't concern the report download directly and until the rows=-1 issue is completely resolved and discussed, this should be dropped.